src/DDoSX/DomainClient.php

```diff
@@ -13,7 +14,13 @@ class DomainClient extends BaseClient
      */
     protected $basePath = 'ddosx/';
 
-    protected $requestMap = [];
+    /**
+     * @var array
+     */
+    protected $requestMap = [
```

Comment:
This `requestMap` is now misleading, as someone may think this is for a domain, not an IP (which it now is). Could we have two different arrays with the name in for clarit please? Something like `$domainMap` and `ipMap`?

Reply:
Can the requestMap not be used for both though it's just a list of responses and entity properties

Reply:
Hmm, my gut tells me no because it's mixing concern but happy to tag others in to discuss!

Reply:
Maybe declare another property for the IP map?

Something like `protected $ipMap` that would contain mapping info for an IP address

You can then pass that instead of `$requestMap` when using the `apiToFriendly` and `friendlyToApi` methods

Reply:
ok will add the ipMap